Sources/Build/Toolchain.swift

```diff
@@ -31,6 +31,12 @@ public protocol Toolchain {
     var dynamicLibraryExtension: String { get }
 }
 
+extension Toolchain {
```

Comment:
I mean, why the extension? Computed property is fine but I think we can keep it in the main struct itself.